win32/Demos/win32gui_dialog.py

```diff
@@ -84,7 +84,7 @@ def toparam(self):
                     # LVM_SETITEMW etc.
```

Comment:
> # Note this demo still works with byte strings.  An
> # alternate strategy would be to use unicode natively
> # and use the 'W' version of the messages - eg,

For instance this comment leads me to believe it won't be as simple as changing the encoding, unless it was specific to Python 2 ?